backend/orchestrator/resources/dynamodb.ts

```diff
@@ -0,0 +1,23 @@
+export const PARTITION_KEY = 'pk';
+export const SORT_KEY = 'sk';
+
+export const DeploymentTable = {
```

Comment:
We can leverage https://github.com/getlift/lift/blob/master/docs/database-dynamodb-single-table.md instead of relying on cloudformation